src/modules/partition/gui/EncryptWidget.h

```diff
@@ -38,6 +38,12 @@ class EncryptWidget : public QWidget
     Encryption state() const;
     void setText( const QString& text );
 
+    /**
+     * @brief setFilesystem sets the filesystem name used for password validation
+     * @param fs A QString containing the name of the filesystem
+     */
+    void setFilesystem( const QString& fs );
```

Comment:
This should be `setFilesystemName` then, since it's .. well, maybe `...Label` except the Label is something else again (?).  It's definitely not the Filesystem though, then I'd expect to use an enum from KPMCore.

Reply:
I am converting it to an enum later.  It is probably better to change it here so it expects an enum.